apps/src/templates/teacherDashboard/AssignmentVersionSelector.story.jsx

```diff
@@ -3,40 +3,30 @@ import {action} from '@storybook/addon-actions';
 import AssignmentVersionSelector from './AssignmentVersionSelector';
 import {courseOfferings} from '@cdo/apps/templates/teacherDashboard/teacherDashboardTestHelpers';
 
+export default {
+  title: 'AssignmentVersionSelector',
+  component: AssignmentVersionSelector
+};
+
 const styles = {
```

Comment:
A little out of the scope of this PR, but something we could consider: It seems like a 'code smell' that our UI styleguide has this internal style, but since the component itself relies on a style being passed in, it also makes sense? I wonder if it's worth adding a second story with different padding here to showcase that the dropdown styling is not 'built-in' to the AssignmentVersionSelector and can be adjusted as needed?

Reply:
I added another rendering of the enabled version without any styling! I hope this showcases the (totally minor) difference.